extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -21,22 +21,23 @@ const Playlist = memo( ( { playerId, tracks, selectTrack, currentTrack, playerSt
 			aria-labelledby={ `jetpack-podcast-player__tracklist-title--${ playerId }` }
 			aria-describedby={ `jetpack-podcast-player__tracklist-description--${ playerId }` }
 		>
-			{ tracks.map( ( track, index ) => {
-				const isActive = currentTrack === index;
+			{ tracks?.length > 1 &&
+				tracks.map( ( track, index ) => {
+					const isActive = currentTrack === index;
 
-				return (
-					<Track
-						key={ track.id }
-						index={ index }
-						track={ track }
-						selectTrack={ selectTrack }
-						isActive={ isActive }
-						isPlaying={ isActive && playerState === STATE_PLAYING }
-						isError={ isActive && playerState === STATE_ERROR }
-						colors={ colors }
-					/>
-				);
-			} ) }
+					return (
+						<Track
+							key={ track.id }
+							index={ index }
+							track={ track }
+							selectTrack={ selectTrack }
+							isActive={ isActive }
+							isPlaying={ isActive && playerState === STATE_PLAYING }
+							isError={ isActive && playerState === STATE_ERROR }
+							colors={ colors }
+						/>
+					);
+				} ) }
```

Comment:
How come we get changes showing here for tabs vs spaces? Is this something wrong with your editor config?

Reply:
AFAIR this was done by the precommit hook. It's indenting the whole block by an additional level because of the new condition added in [line 24](https://github.com/Automattic/jetpack/pull/16054/files/502e3c73134c63cbb6d983a48cd84285ad8f4970#diff-f7d246f05f2a412fe04fa6d020c8e66fR24).